corehq/apps/user_importer/importer.py

```diff
@@ -340,7 +342,7 @@ def create_or_update_users_and_groups(domain, user_specs, group_memoizer=None, u
 
                     # note: explicitly not including "None" here because that's the default value if not set.
                     # False means it was set explicitly to that value
-                    if is_account_confirmed is False:
+                    if is_account_confirmed is False and not web_user:
```

Comment:
The assumption is that for users with the web user set, all access will happen via the web user, the mobile user is a convenience for data entry, but will never be authenticated as. Additionally, since the mappings are 1 to 1, we also don't want to send two confirmations, so whichever they choose, we will do for the web user.